### ISSUE TITLE
VIVO-1709: Verify installation documentation for Scholars react

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#3299]
[Andrew Woods](https://github.com/awoods) (Migrated from [VIVO-1709](https://vivo-project.atlassian.net/browse/VIVO-1709)) said:

Given an running scholars-discovery, this task is to verify the installation documentation for Scholars react:
- https://github.com/vivo-community/scholars-react/blob/master/README.md

Success for this ticket is for two people to have successfully installed discovery/react based on the project README files.

It may be the case that documentation needs to be updated in order for the above success criteria to be met. Such documentation updates can either be completed in the scope of this ticket or a new ticket.

**Comment by chenejac** [comment]
This issue depends on #3308
